upcloud/gateway.go

```diff
@@ -45,6 +45,12 @@ type Gateway struct {
 	Routers          []GatewayRouter         `json:"routers,omitempty"`
 	CreatedAt        time.Time               `json:"created_at,omitempty"`
 	UpdatedAt        time.Time               `json:"updated_at,omitempty"`
+	Addresses        []GateWayAdress         `json:"addresses,omitempty"`
```

Comment:
```suggestion
	Addresses        []GatewayAddress         `json:"addresses,omitempty"`
```

To align with the above `GatewayRouter` parameter